tests/acceptance/08_commands/01_modules/module-array-allows-slash.cf

```diff
@@ -10,10 +10,6 @@ bundle agent test
   meta:
       "description" string => "Test that arrays defined by modules can contain slashes just like classic arrays.";
```

Comment:
Can you add the ticket as a promisee/stakeolder?
```
"description" -> { "CFE-2478" }
  string => "Test that arrays defined by modules can contain slashes just like classic arrays.";
```

Reply:
Done @nickanderson